concourse/scripts/create-cf.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+SCRIPT_DIR=$(cd "$(dirname $0)" && pwd)
+ATC_URL=${ATC_URL:-"http://192.168.100.4:8080"}
+FLY_TARGET=${FLY_TARGET:-$ATC_URL}
+
+env=${DEPLOY_ENV-$1}
+pipeline="deploy-cloudfoundry"
```

Comment:
In the rest of pipelines, the name of the pipeline file definition and the script to upload it matches with the name of the pipeline (see [create-deployer](https://github.com/alphagov/paas-cf/blob/110234556_cf_deploy/concourse/scripts/create-deployer.sh#L10--1) or [create-microbosh](https://github.com/alphagov/paas-cf/blob/110234556_cf_deploy/concourse/scripts/create-microbosh.sh#L10--1).

Shouldn't we do the same for this pipeline?

In that case, I would suggest rename the file `create-cf.sh` to `deploy-cloudfoundry.sh` and `create-cf.yml` to `deploy-cloudfoundry.yml`.